geonode_mapstore_client/client/js/epics/gnviewer.js

```diff
@@ -144,7 +148,34 @@ export const gnViewerRequestGeoStoryConfig = (action$) =>
                 return Observable.empty();
             });
         });
-
+export const gnViewerRequestNewGeoStoryConfig = (action$, { getState = () => {}}) =>
+    action$.ofType(REQUEST_NEW_GEOSTORY_CONFIG)
+        .switchMap(() => {
+            return Observable.defer(() => axios.all([
+                getNewGeoStoryConfig()
+            ])).switchMap((response) => {
```

Comment:
we don't need axios.all in this case it could be:
`Observable.defer(() => getNewGeoStoryConfig()).switchMap((gnGeoStory) => `